packages/metrics/mempool.go

```diff
@@ -0,0 +1,66 @@
+package metrics
+
+import (
+	"github.com/iotaledger/hive.go/logger"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type MempoolMetrics interface {
+	NewOffLedgerRequest()
+	NewOnLedgerRequest()
+	ProcessRequest()
+}
+
+func (c *chainMetrics) NewOffLedgerRequest() {
+	c.metrics.offLedgerRequestCounter.With(prometheus.Labels{"chain": c.chainID.String()}).Inc()
+}
+
+func (c *chainMetrics) NewOnLedgerRequest() {
+	c.metrics.onLedgerRequestCounter.With(prometheus.Labels{"chain": c.chainID.String()}).Inc()
+}
+
+func (c *chainMetrics) ProcessRequest() {
+	c.metrics.processedRequestCounter.With(prometheus.Labels{"chain": c.chainID.String()}).Inc()
+}
+
+func (m *Metrics) registerMempoolMetrics() {
+	m.log.Info("Registering mempool metrics to prometheus")
+	m.offLedgerRequestCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "wasp_off_ledger_request_counter",
+		Help: "Number of ledger requests made to chains",
+	}, []string{"chain"})
```

Comment:
typo: "Number of off-ledger requests made to chains"

Reply:
Actually, should be a single chain, right? 😄 
```suggestion
		Help: "Number of off-ledger requests made to the chain",
```